package.json

```
@@ -1,55 +1,46 @@
-  "##": "All Rights Reserved.",
   "name": "@openinf/docker-fisher",
   "private": "true",
   "description": "The OpenINF Docker container construction yard",
-  "packageManager": "pnpm@8.15.1",
-  "engines": {
-    "node": "20.11.0",
-    "pnpm": "8.15.1"
-  },
   "homepage": "open.inf.is",
   "repository": "OpenINF/docker-fisher",
   "license": "MIT OR Apache-2.0 OR BlueOak-1.0.0",
   "author": "The OpenINF Authors",
   "scripts": {
     "format:all": "for i in build/tasks/format/*.mjs; do node \"$i\"; done",
+    "format:bash": "node build/tasks/format/format-bash.mjs",
     "format:css": "node build/tasks/format/format-css.mjs",
+    "format:dockerfile": "node build/tasks/format/format-dockerfile.mjs",
+    "format:fish": "node build/tasks/format/format-fish.mjs",
     "format:html": "node build/tasks/format/format-html.mjs",
     "format:js": "node build/tasks/format/format-js.mjs",
     "format:json": "node build/tasks/format/format-json.mjs",
     "format:md": "node build/tasks/format/format-md.mjs",
     "format:svg": "node build/tasks/format/format-svg.mjs",
     "format:yaml": "node build/tasks/format/format-yaml.mjs",
-    "format:fish": "node build/tasks/format/format-fish.mjs",
-    "format:bash": "node build/tasks/format/format-bash.mjs",
-    "format:dockerfile": "node build/tasks/format/format-dockerfile.mjs",
     "test": "npm run verify:all",
     "verify:all": "for i in build/tasks/verify/*.mjs; do node \"$i\"; done",
+    "verify:bash": "node build/tasks/verify/verify-bash.mjs",
     "verify:css": "node build/tasks/verify/verify-css.mjs",
+    "verify:dockerfile": "node build/tasks/verify/verify-dockerfile.mjs",
     "verify:ec-harmony": "node build/tasks/verify/verify-ec-harmony.mjs",
-    "verify:sc-harmony": "node build/tasks/verify/verify-sc-harmony.mjs",
+    "verify:fish": "node build/tasks/verify/verify-fish.mjs",
     "verify:html": "node build/tasks/verify/verify-html.mjs",
     "verify:js": "node build/tasks/verify/verify-js.mjs",
     "verify:json": "node build/tasks/verify/verify-json.mjs",
     "verify:md": "node build/tasks/verify/verify-md.mjs",
+    "verify:sc-harmony": "node build/tasks/verify/verify-sc-harmony.mjs",
     "verify:svg": "node build/tasks/verify/verify-svg.mjs",
-    "verify:yaml": "node build/tasks/verify/verify-yaml.mjs",
-    "verify:fish": "node build/tasks/verify/verify-fish.mjs",
-    "verify:bash": "node build/tasks/verify/verify-bash.mjs",
-    "verify:dockerfile": "node build/tasks/verify/verify-dockerfile.mjs"
+    "verify:yaml": "node build/tasks/verify/verify-yaml.mjs"
   },
   "devDependencies": {
     "@babel/core": "7.23.9",
     "@babel/eslint-parser": "7.23.10",
     "@babel/plugin-syntax-jsx": "7.23.3",
     "@babel/plugin-syntax-top-level-await": "7.14.5",
     "@babel/preset-env": "7.23.9",
-    "@yarnpkg/shell": "4.0.0",
-    "cspell": "8.3.2",
+    "@openinf/util-text": "OpenINF/openinf-util-text#main",
```

```suggestion
    "@openinf/util-text": "OpenINF/openinf-util-text#main",
```

we still need to build this; thinking to put the output in another branch